src/bundle/Resources/config/bazinga_js_translation.yml

```diff
@@ -0,0 +1,6 @@
+active_domains:
+    - 'role'
+    - 'trash'
+    - 'dashboard'
+    - 'ezrepoforms_content'
+    - 'universal_discovery_widget'
```

Comment:
I believe our intention was to only have one domain for UDW `universal_discovery_widget` with all titles etc. @dew326 @Nattfarinn @adamwojs